agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/BeforeAgentInstaller.java

```diff
@@ -69,10 +76,11 @@
 import org.slf4j.LoggerFactory;
 
 import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class BeforeAgentInstaller {
 
-    private static Logger startupLogger = LoggerFactory.getLogger("com.microsoft.applicationinsights.agent");
+    private static final Logger startupLogger = LoggerFactory.getLogger("com.microsoft.applicationinsights.agent");
```

Comment:
when i tested this change with spring-petclinic app, startupLogger didn't output to console even though i set it to info level... it does when I used "system.out.println".  Do you know what might be the cause?  Everywhere else, we pass Class name the LoggerFactor.getLogger.. here we use part of the package name... is there a reason for that? don't want to disclose class name in the log?